models/sources/src_history_operations.yml

```diff
@@ -9,7 +9,7 @@ sources:
         tests:
           - dbt_utils.recency:
               datepart: hour
-              field: closed_at
+              field: cast(closed_at as datetime)
```

Comment:
does it matter that the partition column for this table is actually `batch_run_date`?

Reply:
Hmm you're right it doesn't matter